lib/client.ts

```diff
@@ -303,6 +318,13 @@ export default class Client {
     return ensureJSON(res);
   }
 
+  public async getNarrowcastProgress(requestId: string) {
```

Comment:
Add a return type Promise<Types.NarrowcastProgressResponse> to function definition
```suggestion
  public async getNarrowcastProgress(requestId: string): Promise<Types.NarrowcastProgressResponse> {
```